reference/configuration/framework.rst

```diff
@@ -1315,6 +1315,7 @@ The possible values for this option are:
 
 * ``null``, use it to disable this protection. Same behavior as in older Symfony
   versions.
+* ``'none'`` (or the ``Cookie::SAMESITE_NONE`` constant), use it to allow sending of cookies when the HTTP request originated from a different domain (Previously this was the default behavior of null, but in newer browsers 'lax' would be applied when the header has not been set)
```

Comment:
nice add, but i think you should break line in the sentence to make short (see previous for example)

Reply:
Fixed 👍